feature_engine/selection/single_feature_performance.py

```diff
@@ -32,10 +33,8 @@
 from feature_engine.selection.base_selector import BaseSelector
 from feature_engine.tags import _return_tags
 
-from .base_selection_functions import (
-    _select_numerical_variables,
-    single_feature_performance,
-)
+from .base_selection_functions import (_select_numerical_variables,
```

Comment:
sorry to bother with this, could you please pass the imports to the following lines, so we keep it consistent?

Reply:
I put them back, sorry for having linting confusion again, isort doesn't work properly in my environment

Reply:
thank you!